core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -1201,7 +1202,7 @@ private[spark] object SparkSubmitUtils {
    */
```

Comment:
@AngersZhuuuu  FYI you missed updating the `@return` Scaladoc tag here, as well as the description which explicitly mentions a comma-delimited list. Would you mind submitting a follow-on to update?